docs/hub/mlx.md

```diff
@@ -0,0 +1,100 @@
+# Using MLX at Hugging Face
+
+[MLX]([https://github.com/MaartenGr/BERTopic](https://github.com/ml-explore/mlx)) is a model training and serving framework on Apple Silicon made by Apple Research.
+
+It comes with variety of examples:
+
+- [Transformer language model](https://github.com/ml-explore/mlx-examples/tree/main/transformer_lm) training.
```

Comment:
I'd add reference to these two first:

1. https://github.com/ml-explore/mlx-examples/tree/main/llms/mlx_lm
2. https://github.com/ml-explore/mlx-examples/tree/main/llms/gguf_llm